classes/03class/exercises/c03-docker02/starkmatt/c03-docker02.txt

```diff
@@ -7,10 +7,15 @@
 
```

Comment:
This PR should be about `c03-aws01` only.

Can you remove `c03-docker02.txt`? 
Thanks!